pyproject.toml

```diff
@@ -8,14 +8,16 @@ license = "MIT"
 [tool.poetry.dependencies]
 python = "^3.9, <3.11"  # Tensorflow requires <3.11
 docker = "^5.0.3"
-gym = "^0.23.1"
 pydantic = "^1.9.2"
 ray = {extras = ["rllib"], version = "^2.0.0"}
 fancyflags = "^1.1"
 absl-py = "^1.3.0"
-wandb = "^0.13.4"
+wandb = "^0.13.11"
 torch = "^1.13.0"
 dm-tree = "^0.1.7"
+scikit-image = "^0.20.0"
```

Comment:
Where do use `scikit-image` and `protobuf`? Wondering why we need to specify these top-level if they're just transitive dependencies of other deps (which hopefully specify their own requirements).

But we do depend on `gymnasium` at the top-level, i.e. because we directly import it in our own code, I wouldn't depend on it transitively through another dep like ray. I think we should list it here ourselves.

Reply:
Protobuf downgrade was because of a protobuf bug in wandb: https://github.com/wandb/wandb/issues/1862

Scikit-image was because of some random untrackable dependency bug. I'm not sure, you can try remove it in the future if it doesn't break dependencies.